packages/core/test/data-query/builder/where-clause.spec.ts

```diff
@@ -75,6 +78,7 @@ describe('Test data query builder > where clause', () => {
       const builder = new DataQueryBuilder({
         statement: 'select * from orders',
         dataSource: stubDataSource,
+        bindParams: sinon.stubInterface<BindParameters>(),
```

Comment:
NIT: I'd prefer to create a helper function like `createMockBuild` to prevent potential changes in the future.